Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Editor/SpectatorViewWindowBase.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.Editor
+{
+    public class SpectatorViewWindowBase<TWindow> : EditorWindow where TWindow : SpectatorViewWindowBase<TWindow>
```

Comment:
Could we name this more generic, as this is a pretty awesome WindowBase in general for Editor

Reply:
Renamed this to EditorWindowBase, open to other name suggestions